packages/application-config/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
If you're unfamiliar with JSON schema, you can have a look at their documentation: https://json-schema.org/learn/getting-started-step-by-step.html